themes/digital.gov/layouts/shortcodes/hs-form-subscribe.html

```diff
@@ -1,10 +1,2 @@
-<!--[if lte IE 8]>
-<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
-<![endif]-->
-<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
-<script>
-  hbspt.forms.create({
-	portalId: "1962994",
-	formId: "f8c8ff73-dbc9-4242-90e4-e8cbcee02f6a"
-});
-</script>
+<h3>Subscribe to the Digital.gov Newsletter</h3>
+<button class="subscribe-button usa-button usa-button--outline">Go to subscribe</button>
```

Comment:
Let's add `type="button"` so screen readers don't confuse this for a form submit.

```suggestion
<button type="button" class="subscribe-button usa-button usa-button--outline">Go to subscribe</button>
```

Reply:
Good catch, `type="button"` is now added to the subscribe button.